adapter/src/dbt/adapters/fal_experimental/support/trino.py

```diff
@@ -17,10 +17,7 @@ def create_engine(adapter: BaseAdapter) -> Any:
     )
     return sqlalchemy.create_engine(url, connect_args=connect_args)
 
-def _build_connect_args(credentials: TrinoCredentials) -> Optional[Dict[str, Any]]:
-    if isinstance(credentials, TrinoNoneCredentials):
```

Comment:
ATL: why we no longer need to handle this case?

Reply:
`TrinoNoneCredentials.trino_auth()` returns None and we need to pass http_scheme.
```
return {
        "auth": credentials.trino_auth(),
        "http_scheme": credentials.http_scheme.value
}
```